cli/worker.rs

```diff
@@ -82,7 +78,26 @@ pub trait HasNodeSpecifierChecker: Send + Sync {
   fn has_node_specifier(&self) -> bool;
 }
 
-#[derive(Clone)]
+#[async_trait::async_trait(?Send)]
+pub trait HmrRunner: Send + Sync {
+  fn setup(&mut self, session: deno_core::LocalInspectorSession);
```

Comment:
I think this would be cleaner if we provided a factory for HmrRunner and CoverageCollector instead of having a "half configured" pre-setup state. We'll probably get bitten by spurious `unwrap()` panicks down the road unless we enforce this via the type system.

ie:

`Box<dyn FnOnce(LocalInspectorSession)> -> Box<dyn CoverageCollector>`

Reply:
Good point. I've added `create_hmr_runner` and `create_coverage_collector` creation callbacks.